### PR TITLE
1.7.x

### PR DESCRIPTION
**Pull request by lgarciaaco**
This PR adds yet another file (`resources/fuse-online-template-oh.yml `) which is exclusively used by syndesis-operator when passed the `-v` option (when installed from the operatorhub). https://github.com/syndesisio/syndesis/pull/6189 depends on this being merged. On top of that:

- Add `insecure: {{ Docker.Registry.Insecure }}` to fuse-online-image-streams.yml template in an attempt to make the installation work with brew .. it doesn't.

- Prepare future releases to pull from quay.io, where the productized images for testing are also uploaded, and there is not problem with HTTP to HTTPS.

- `docker run ...` bits in the release.sh doesn't work for me, I am not sure if I am an isolated case, if yes, I am happy to remove my work around.

### CODE REVIEW

**Comment by lgarciaaco** [comment]
Fixes https://github.com/syndesisio/syndesis/issues/5925

File: templates/fuse-online-image-streams.yml [hunk 1]
```diff
@@ -119,9 +125,10 @@ items:
     tags:
       - from:
           kind: DockerImage
-          name: {{ Docker.Registry }}/fuse7-tech-preview/fuse-postgres-exporter:{{ Tags.Online.PostgresExporter }}
+          name: {{ Docker.Registry }}/{{ Docker.Image.Repository }}/fuse-postgres-exporter:{{ Tags.Online.PostgresExporter }}
```
Comment (heiko-braun) [inline templates/fuse-online-image-streams.yml]: How is this changed for GA images?

File: templates/fuse-online-image-streams.yml [hunk 2]
```diff
@@ -35,6 +36,7 @@ items:
         name: {{ Docker.Registry }}/{{ Docker.Image.Repository }}/fuse-ignite-ui:{{ Tags.Online.Ui }}
       importPolicy:
         scheduled: true
+        insecure: {{ Docker.Registry.Insecure }}
```
Comment (heiko-braun) [inline templates/fuse-online-image-streams.yml]: Why was this added?

File: fuse_online_config.sh [hunk 3]
```diff
@@ -20,12 +20,17 @@ tag_java_base_image="1.4-6"
 tag_komodo="1.4-4"
 
 # Docker repository for productised images
-repository="fuse7"
+#repository="fuse7"
 
 # Test:
-registry="brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888"
+registry="quay.io"
```
Comment (heiko-braun) [inline fuse_online_config.sh]: quay is fine, under two conditons:

a) everybody in the fuse team can access that repo (credentials)
b) we control what images get pushed and when

Comment (heiko-braun) [reply fuse_online_config.sh]: A shared google doc would help. Does QE have docs we can refer to?

**Comment by heiko-braun** [comment]
On a general note, please quash your commits in this PR

**Comment by lgarciaaco** [comment]
I will do some cleanup and reopen
